JASP-Engine/JASP/R/contingencytablesbayesian.R

```diff
@@ -1306,168 +1306,168 @@ ContingencyTablesBayesian <- function(dataset, options, perform, callback, ...)
 		} else if (options$bayesFactorType == "LogBF10") {
 			bfLabel <-"Log\u2009(\u2009BF\u2081\u2080\u2009) hypergeometric"
 		}
-		
+
 		sampleType <- "hypergeom"
 		fixedMargin <- NULL
-		
+
 	} else {
-	
+
 		stop("wtf?")
```

Comment:
@TimKDJ are you responsible for this? :P

Reply:
Hehe no, but I've seen it before in the code as well ^^